hypothesisfuzzer/fuzzing/repo_fuzzer.py

```diff
@@ -222,29 +219,40 @@ def _fuzz_task(self):
         while getattr(self._current_fuzzing_task, "running", True):
 
             logger.info('Fuzzing iteration %s.', self._iterations)
-            subprocess.run(['venv/bin/pytest',
-                            '-m hypothesis',
-                            '--hypothesis-server',
-                            '--hypothesis-output=' + self.name + '.json'],
-                           universal_newlines=True,
-                           stdout=subprocess.PIPE,
-                           cwd=self._project_root)
+
+            result = subprocess.run(['venv/bin/pytest', '-m hypothesis',
+                                     '--hypothesis-server',
+                                     '--hypothesis-output=' + self.name
+                                     + '.json'],
+                                    universal_newlines=True,
+                                    stdout=subprocess.PIPE,
+                                    cwd=self._project_root)
+
+            if result.returncode == 0:
+                pass
+            elif result.returncode == 1:
+                logger.info('Some tests failed.')
+            else:
+                self._ready = False
+                self._currently_fuzzing = False
+                self._status = 'Test execution interrupted by error.'
+                logger.warning('Test execution interrupted by error.')
+                return generic_error('Test execution interrupted by error.')
```

Comment:
Just `return` by itself. `generic_error` is (I think) an HTTP response. It's old and all the errors and exception handling in general need to be refactored and reviewed, but thats for another ticket.